src/compas_rhino/geometry/surfaces/surface.py

```diff
@@ -189,7 +189,8 @@ def curvature_at(self, u, v):
         :class:`~compas.geometry.Vector`
 
         """
-        vector = self.rhino_surface.CurvatureAt(u, v)
+        surface_curvature = self.rhino_surface.CurvatureAt(u, v)
+        vector = surface_curvature.Direction(0)
```

Comment:
wouldn't we rather change it to return a tuple of the two principle curvature directions?
otherwise, to compute the perpendicular direction, the user would have to also get the normal vector and compute an additional cross product...

Reply:
or even return a tuple with the same info as contained in the surface curvature object?

Reply:
The tuple as a return would be an elegant solution, there are many values returned that could be of use

Reply:
would you mind updating the PR with the tuple as output?
@gonzalocasas no?